src/disco/addonManager.js

```diff
@@ -36,3 +42,23 @@ export function uninstall(guid, {_mozAddonManager = window.navigator.mozAddonMan
       return;
     });
 }
+
+export function addChangeListeners(callback, mozAddonManager) {
```

Comment:
I felt this was more accurate since it says what it does more closely. For example if this was addChangeListener you might expect it to only register one event where this doesn't it, registers listeners for multiple events.